src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/GTKMediaPlayer.cs

```diff
@@ -0,0 +1,461 @@
+#nullable enable
+
+using Windows.UI.Core;
+using LibVLCSharp.GTK;
+using LibVLCSharp.Shared;
+using Windows.UI;
+using Windows.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+using System;
+using Windows.UI.Xaml;
+using System.Threading;
+using System.Linq;
+using Windows.UI.Notifications;
+using System.Globalization;
+using System.Collections.Immutable;
+using System.IO;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+using Windows.Foundation;
+using Windows.UI.Xaml.Media;
+using Point = Windows.Foundation.Point;
+using Pango;
+using Uno.UI.Extensions;
+using Windows.Media.Playback;
+using Uno.Logging;
+using Windows.UI.ViewManagement;
+using Windows.Media.Core;
+
+namespace Uno.UI.Media;
+
+public partial class GtkMediaPlayer : Border
```

Comment:
We needed an element that dealt with Childor Content and had height and width control.

Reply:
As you are in Uno, you can inherit from `FrameworkElement` and use the internal `AddChild(UIElement)`. Inheriting from `Border` exposes lot of irrelevant properties, and gives the feeling to the user that he can change the `Child` while doing this would break everything.

Reply:
Done. Updated the border for FrameworkElement.
Thanks for the review.